### DISCUSSION TITLE
Configure Supabase database settings programmatically?

### DISCUSSION THREAD

**Discussion by SamuelCivalgo** [supabase/supabase#12295] [Questions] [answered]
Is it possible to setup supabase database config(RLS rules, set table to Realtime enable, etc..) in code instead of the interface?

**Comment by GaryAustin1** [comment, marked as answer]
If RTS means RLS yes.  Several areas of the docs show how.
Most of these examples are using SQL to set up RLS -  https://supabase.com/docs/guides/auth/row-level-security

Realtime setup with SQL shown here - https://supabase.com/docs/guides/realtime/postgres-changes

**Comment by SamuelCivalgo** [comment]
thanks
